src/cljs/main/broadfcui/page/method_repo/method/summary.cljs

```diff
@@ -0,0 +1,103 @@
+        (modals/show-modals
+         state
+         {:deleting?
+          [Redactor {:entity selected-snapshot :config? false :on-delete #(nav/go-to-path :method-repo)}]
+          :sharing?
+          [mca/AgoraPermsEditor
+           {:save-endpoint (endpoints/persist-agora-entity-acl false selected-snapshot)
+            :load-endpoint (endpoints/get-agora-entity-acl false selected-snapshot)
+            :entityType (:entityType selected-snapshot)
+            :entityName (mca/get-ordered-name selected-snapshot)
+            :title (str (:entityType selected-snapshot) " " (mca/get-ordered-name selected-snapshot))}]
+          :editing?
+          [create/CreateMethodDialog
+           {:snapshot selected-snapshot
+            :on-created on-method-created}]
+          :cloning?
+          [create/CreateMethodDialog
+           {:duplicate selected-snapshot
+            :on-created on-method-created}]})
+        [Sticky
+         {:sticky-props {:data-check-every 1
+                         :data-anchor body-id}
+          :contents
+          [:div {:style {:width 270}}
+           (when owner?
+             [SidebarButton
+              {:style :light :color :button-primary
+               :text "Permissions..." :icon :settings :margin :bottom
+               :onClick #(swap! state assoc :sharing? true)}])
+           (when owner?
+             [SidebarButton
+              {:style :light :color :button-primary
+               :text "Edit..." :icon :edit :margin :bottom
+               :onClick #(swap! state assoc :editing? true)}])
+           [SidebarButton
+            {:style :light :color :button-primary
+             :text "Clone..." :icon :clone :margin :bottom
+             :onClick #(swap! state assoc :cloning? true)}]
+           (when owner?
+             [SidebarButton
+              {:style :light :color :exception-state
+               :text "Redact" :icon :delete :margin :bottom
+               :onClick #(swap! state assoc :deleting? true)}])]}]]))
+   :-render-main
+   (fn [{:keys [props locals]}]
+     (let [{:keys [synopsis managers createDate documentation]} (:selected-snapshot props)
+           {:keys [body-id]} @locals]
+       [:div {:style {:flex "1 1 auto" :overflow "hidden"} :id body-id}
+        (style/create-summary-block "Synopsis" synopsis)
+        [:div {:style {:display "flex"}}
+         (style/create-summary-block (str "Method Owner" (when (> (count managers) 1) "s"))
+                                     (string/join ", " managers))
+         (style/create-summary-block "Created" (common/format-date createDate))]
+
+        [Collapse
+         {:style {:marginBottom "2rem"}
+          :title (style/create-subsection-header "Documentation")
+          :contents
+          [:div {:style {:marginTop "1rem" :fontSize "90%" :lineHeight 1.5}}
+           (if (not-empty documentation)
+             [MarkdownView {:text documentation}]
+             [:em {} "No documentation provided"])]}]]))
```

Comment:
1. I find it visually weird that - I assume because of the `Collapse` - the documentation section is nudged to the right a bit compared to synopsis and method owner.

2. Love the markdown support! This resolves GAWB-1970 - could you make sure to pull that ticket in, please?

Reply:
The one thing I meant to do along with this was to put the markdown editor in the create/clone method modal. Should be quite simple.

Reply:
(ok, back to vacation now)